### ISSUE TITLE
Multi-select drop downs in advanced search keyboard trap with JAWS

### ISSUE THREAD

**Issue by ajkiessl** [psu-libraries/psulib_blacklight#1175]
Once inside the drop down, pressing escape does not remove the user from the drop down with JAWS.  This traps the user inside of the drop down.  The user is able to escape if they turn off the PC cursor.  

Attempts to update blacklight and bootstrap-select did not fix this.
